### ISSUE TITLE
Ubuntu 14.04 dependency libgit2-0

### ISSUE THREAD

**Issue by xlvdk** [kbenzie/git-prompt#2]
It may be good to add libgit2-0 as a dependecy for Ubuntu users to install. That fixed my only installation issue :D

**Comment by kbenzie** [comment]
Yeah I'm planning adding the option to use the system version of `libgit2` for building and runtime as well as being able to build and install it as part of the project. When I tested it using the CMake install target `libgit2` was installed as well as `git-prompt` so I didn't see this problem.
